app/server-routes/src/main/scala/org/bitcoins/server/routes/BitcoinSRunner.scala

```diff
@@ -17,6 +17,8 @@ trait BitcoinSRunner extends BitcoinSLogger {
 
   def actorSystemName: String
 
+  def defaultRpcPort: Int
```

Comment:
So in my opinion this should be read from the default `reference.conf` file. 

We have settings for these in these files I believe 

`bitcoin-s.server.rpcport`

`bitcoin-s.oracle.rpcport`

I'm not seeing the latter in the default configuration file, but we should add it. Basically any project that is a server project should have the ability to set the rpc port 

https://bitcoin-s.org/docs/config/configuration#example-configuration-file

Reply:
We still need a default port hard coded in, I think the current way is better.

Reply:
I guess in my mind it would be hardcoded in the reference.conf file and then read as part of the `AppConfig`. In general i think it's worth trying to preserve the property that every configuration 

1. Lives in a `application.conf`/`reference.conf` file
2. Is passed in explicitly with a command line param (`--rpcport`)

With `bitcoin-s-cli` it isn't possible to achieve (1), so I'm more amenably to hard coding there, but I would like to try and preserve this property with the server. That way when we show example confs on the website, users can understand where the configuration is coming from rather than having to dig in source code.